scripts/protocgen.sh

```diff
@@ -8,7 +8,7 @@ protoc_gen_gocosmos() {
     return 1
   fi
 
-  go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos@latest 2>/dev/null
+  go install github.com/regen-network/cosmos-proto/protoc-gen-gocosmos@latest 2>/dev/null
```

Comment:
still unsure why, but this is breaking `make proto-gen` for me. Can someone run this on their machine and see if its just my local machine?

Reply:
[10ad4f8](https://github.com/celestiaorg/celestia-app/pull/282/commits/10ad4f87d02fefe2b6ed9fb6c0fbb8d09ecb1183)